docs/reference/dqm.md

```diff
@@ -0,0 +1,79 @@
+# Data Quality Monitoring
+
+Data Quality Monitoring (DQM) is a Feast module aimed to help users to validate their data with the user-curated set of rules.
+Validation could be applied during:
+* Historical retrieval (training dataset generation)
+* [planned] Writing features into an online store
+* [planned] Reading features from an online store
+
+Its goal is to address several complex data problems, namely:
+* Data Consistency - new training dataset could be significantly different from the previous, which might require a change in model architecture.
```

Comment:
```suggestion
* Data consistency - new training datasets can be significantly different from previous datasets. This might require a change in model architecture.
```